src/OrchardCore.Modules/OrchardCore.Users/Startup.cs

```diff
@@ -90,8 +82,12 @@ public override void ConfigureServices(IServiceCollection services)
             services.ConfigureApplicationCookie(options =>
             {
                 options.Cookie.Name = "orchauth_" + _tenantName;
-                // Use 'PathBase' if e.g under a virtual folder.
-                options.Cookie.Path = _pathBase + _tenantPrefix;
+
+                // Don't set the cookie builder 'Path' so that it uses the 'IAuthenticationFeature' value
+                // set by the pipeline and comming from the request 'PathBase' which already ends with the
+                // tenant prefix but may also start by a path related e.g to a virtual folder.
+                // options.Cookie.Path = _tenantPrefix;
```

Comment:
Should that be removed? (just this line, not the precious comment)

Reply:
Okay